compiler/fir/resolve/src/org/jetbrains/kotlin/fir/resolve/inference/FirBuilderInferenceSession.kt

```diff
@@ -269,9 +268,6 @@ class FirStubTypeTransformer(
         substitutor.substituteOrNull(resolvedTypeRef.type)?.let {
             resolvedTypeRef.withReplacedConeType(it)
         } ?: resolvedTypeRef
-
```

Comment:
Can you please elaborate this change? This transformer has nothing to do with regular body resolve transformers and needed only for builder inference

Reply:
That change has no effect, the override does the same thing as `transformElement`. I removed it so that it doesn't appear in uses of `transformArguments`, which is what I was looking for.

Reply:
Ok. It's worth to extract this change to separate commit

Reply:
done